core-services/prow/02_config/openshift/pagerduty-short-circuiter/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/pagerduty-short-circuiter
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/pagerduty-short-circuiter
+  review_acts_as_lgtm: true
+plugins:
+  openshift/pagerduty-short-circuiter:
+    plugins:
+    - approve
```

Comment:
For `lgtm` and `approve` to be of any use, you also need to configure automated merging (see the `tide` config in other repos' `_prowconfig.yaml`